tests/farmer_harvester/test_filter_prefix_bits.py

```diff
@@ -69,15 +69,10 @@ def state_has_changed() -> bool:
         signage_point_index=uint8(2),
         filter_prefix_bits=filter_prefix_bits,
     )
-    passed = False
     await farmer_api.new_signage_point(sp)
     await time_out_assert(5, state_has_changed, True)
     # We're intercepting the harvester's state changes as we're expecting
-    # a farming_info one. eligible_plots are what passed the plot filter
-    if (
-        state_change == "farming_info"
-        and state_change_data is not None
-        and state_change_data.get("eligible_plots") == 1
-    ):
-        passed = True
-    assert passed == should_pass
+    # a farming_info one.
```

Comment:
```suggestion
    # a farming_info.
```

I think the `one` doesn't really make sense there either?

Reply:
"We're intercepting the harvester's state changes as we're expecting a farming_info one." one here is referring to one state change from many that can theoretically get intercepted from the harvester (and that one state change is of type farming_info). Does it make sense?

Reply:
Seems rather confusing then just saying "we expect a farming_info" but with the explanation it makes more sense now :)